### ISSUE TITLE
Mappo algorithm can not reach the results in the paper？

### ISSUE THREAD

**Issue by BITminicc** [uoe-agents/epymarl#9]
Awesome work for extending the original pymarl! But my experimental results fail to meet the return or battle_won described in the paper using MAPPO algorithm. I have adjusted some parameters according to the chapter I(selected hyperparameters) in the paper, but it didn't work. Is there any detail I ignored that can be adjusted and achieve the result in the paper, especially for SMAC env (MMM2 and corridor).

**Comment by papoudakis** [comment]
Hi,
A quick note: We ran on-policy algorithms, such as MAPPO, for 40M timesteps, and off-policy for 4M timesteps. 
In the plots presented in the paper, we use the x-axis goes up to 4M timesteps in order to be able to compare visually with off-policy algorithms. So, is it possible that you ran MAPPO for only 4M timesteps?

**Comment by BITminicc** [comment]
> Hi, A quick note: We ran on-policy algorithms, such as MAPPO, for 40M timesteps, and off-policy for 4M timesteps. In the plots presented in the paper, the x-axis goes up to 4M timesteps in order to be able to compare visually with off-policy algorithms. So, is it possible that you ran MAPPO for only 4M timesteps?

Thank you for your reply. I reread the 4.1 evaluation protocol in the paper, so can I understand it as setting t_max to 40M will get better results instead of zero win-rates?

**Comment by papoudakis** [comment]
Yes. On-policy algorithms should be trained for 40M timesteps. I will the issue open for now. Please let us know if this does not solve the problem.

**Comment by BITminicc** [comment]
This change worked for mappo with parameter sharing in SMAC tasks. (MMM2) 
I'm particularly grateful to you for your timely help.
